src/lidar/read_rp_lidar.h

```diff
@@ -7,10 +7,19 @@
 
 namespace lidar {
 
+/**
+ * @brief Startsup RPLidar with default settings
+ */
 bool initializeLidar();
```

Comment:
This function prototype no longer matches the signature for the implementation of `initializeLidar` in `read_rp_lidar.cpp`. I'd recommend you change this line to `bool initializeLidar(double max_dist=16);` and change the implementation to be `bool initializeLidar(double max_dist) {...}`